torchci/components/benchmark/compilers/common.tsx

```diff
@@ -52,3 +52,8 @@ export const HELP_LINK =
   "https://pytorch.org/docs/main/torch.compiler_performance_dashboard.html";
 
 export const DTYPES = ["amp", "float16", "bfloat16", "quant"];
+
+export const DEFAULT_DEVICE_NAME = "cuda";
+// TODO (huydhn): there is a way to avoid hard-coding dtypes and devices like how
+// the LLM micro-benchmark page is implemented
+export const DEVICES = ["cuda", "cpu-x86"];
```

Comment:
@huydhn , can we update cpu-x86 to cpu_x86 here? I have triggered a few manual cpu_x86 runs and wanted to check with your UI update.

Reply:
Also, I plan to add a few more, like `cpu_aarch64` and `cuda_A10g`.

Reply:
Updated.  It requires more changes than I expect to pass the device name around.  I have also added `cpu_aarch64` in anticipation of https://github.com/pytorch/pytorch/pull/131729.

Here is the preview snapshot of cpu_x86 https://torchci-git-fork-huydhn-add-cpu-device-pt2-96f356-fbopensource.vercel.app/benchmark/compilers?dashboard=torchinductor&startTime=Thu%2C%2018%20Jul%202024%2007%3A43%3A49%20GMT&stopTime=Thu%2C%2025%20Jul%202024%2007%3A43%3A49%20GMT&granularity=hour&suite=torchbench&mode=inference&dtype=bfloat16&deviceName=cpu%20(x86)&lBranch=main&lCommit=ab609d6aa6851072333c3b1de6f0e72a1db10a51&rBranch=main&rCommit=8aff6caf67472ea0281a75eae40b89d9c433c63b

Reply:
Please take a quick look, I'll land this tmrw.

Reply:
The UI looks good now.